perllib/FixMyStreet/Cobrand/CentralBedfordshire.pm

```diff
@@ -87,6 +87,14 @@ sub disambiguate_location {
 
 sub enter_postcode_text { 'Enter a postcode, street name and area, or check an existing report number' }
 
+sub open311_config {
+    my ($self, $row, $h, $params, $contact) = @_;
+    if ($contact->email =~ /Jadu/) {
+        $params->{multi_photos} = 1;
```

Comment:
I wonder why this isn't on for all Central Beds? Looks like the open311-adapter symology end can cope with multiple URLs being provided.

Reply:
Yeah, does seem fine - have added